utils/shipping-actions.js

```diff
@@ -218,3 +218,20 @@ export function checkShippingPreviewResult(selectors) {
     }
   })
 }
+
+export function interceptAutoCompleteResponse(responseObject) {
+  cy.intercept(
+    'https://maps.googleapis.com/maps/api/place/js/PlaceService.GetPlaceDetails',
+    req => {
+      const searchParams = new URLSearchParams(req.url)
+      const callbackFunctionName = searchParams.get('callback')
+
+      req.reply(`
+${callbackFunctionName} && ${callbackFunctionName}(${JSON.stringify({
+        result: responseObject,
+        status: 'OK',
+      })})
+`)
```

Comment:
Can you tell why do we need the callback function to change the response?

Reply:
because that's how this api works, it returns a javascript snippet which calls the callback passed in the `callback` query string

Reply:
ah that makes sense, ty